freqtrade/optimize/backtesting.py

```diff
@@ -357,8 +357,30 @@ def _get_close_rate(self, sell_row: Tuple, trade: LocalTrade, sell: SellCheckTup
         else:
             return sell_row[OPEN_IDX]
 
+    def _get_adjust_trade_entry_for_candle(self, trade: LocalTrade, row: Tuple
+                                           ) -> LocalTrade:
+
+        current_profit = trade.calc_profit_ratio(row[OPEN_IDX])
+        stake_amount = strategy_safe_wrapper(self.strategy.adjust_trade_position,
```

Comment:
Please put a warning box in the docs advising users that this callback will be called for each `timeframe-detail` candle during backtesting (so if you backtest with `--timeframe 1h --timeframe-detail 1m` - it's going to be called 60 times per hour.

The same happens for exit callbacks - but i find it surprising in this instance.

It can be a feature - BUT you (or rather, users) MUST be aware of this to avoid nasty surprises.

Reply:
I'm adding a documentation warning. This is working as expected as position adjustment is called in live mode also every 5 seconds and the purpose of position adjustment is the ability to react to market movements (not waiting for candle arrival). So in backtesting we are running it also as detailed as possible.

Reply:
As said, it's about being aware / know what to expect :+1: